pages/read/sxcu.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Reads from  '../../components/reads';
+
+const json = {
+    page: 'styling',
+    payload: [{
+        title: 'SXCU Files',
+        description: 'Want to have your Tixte Domain as a Default Uploader?',
+        section: 'how',
+        list: [{
+            id: 1,
+            text: 'Go to the <a href="https://getsharex.com" class="link">ShareX Website</a>.'
+        }, {
+            id: 2,
+            text: 'Download and open the installer.'
+        }, {
+            id: 3,
+            text: 'Go through the Installer.'
```

Comment:
Why is `Installer` uppercase here when it was lowercase in L#15

Reply:
Fixed